roles/gitea/templates/gitea.service.j2

```diff
@@ -0,0 +1,32 @@
+[Unit]
+Description=Gitea (Git with a cup of tea)
+After=syslog.target
+After=network.target
+#After=mysqld.service
```

Comment:
Before merging, the commented out parts need to be sanitized (fixed or removed)

Reply:
This service came from https://docs.gitea.io/en-us/linux-service/. The instructions say "Uncomment any service that needs to be enabled on this host, such as MySQL." What should we do for this?

Reply:
Whichever database goes in `gitea.service` also needs to go in `app.ini`. The options are MySQL, Postgres, MSSQL, and SQLite3. I don't know why memcached and Redis are also in `gitea.service`.

Reply:
[This](https://github.com/iiab/iiab/pull/1242/files#diff-677772bd45695404f35ebdfdb64f4556R220) seems to suggest that gitea will use mysql, so in that case, perhaps the `After=mysqld.service` line needs to be uncommented to avoid potential issues.

Reply:
Might as well; it's the default.

Reply:
Should we leave the Postgres line in there in case the user wants to switch?

Reply:
I would probably remove it. And in its place, leave a comment in the playbook's .service file to the documentation link instead. As it needs changes to the ini file too (iirc).